.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug report
+about: Create a report to help us improve

```

Comment:
It seems `about` has been changed. The `description` key should be used

Reply:
Already fixed